### DISCUSSION TITLE
Roadmap and announcements

### DISCUSSION THREAD

**Discussion by FALLAI-Denis** [IBM/zopeneditor-about#175] [General]
Hi,

We greatly appreciate the work done, and today Z Open Editor is a major component of our DevOps Mainframe solution.

Is there a short / medium term roadmap for the development of Z Open Editor?

How are the priorities decided in taking into account the corrections of anomalies and the integration of changes?

Could the team post content announcements for the next release(s)?

Thanks.

**Comment by phaumer** [comment]
We also appreciate all the great feedback and feature ideas you @FALLAI-Denis and others are providing here. 

As Z Open Editor is also included into commercial offerings such as IBM Wazi Developers we have to follow the general IBM rules for announcing enhancements. One thing we could do is more aggressively close items that we have not plan to work on in the short or mid-term. In the past I left them as there were a lot of good ideas listed that I wanted to preserve and perhaps come back to.

**Reply by FALLAI-Denis** [reply]
Hi @phaumer 

Rather than closing the requests that will not be taken into account in the next iterations, maybe we should switch them to this "Discussions" area, or create (if possible) a new area dedicated to the "long back log", or main new areas with "priority".

**Reply by phaumer** [reply]
Good idea. Let me discuss with the team.

**Comment by FALLAI-Denis** [comment]
For information, today we have two major anomalies which are conditions for not adopting the Z Open Editor solution:
#142 Revocation of the mvs user during the COPYBOOK search for type "mvs" property-group for invalid password
#154 Cobol Language Server - "Full red program" after LS analysis
